targets/frontend/src/components/contributions/answers/Comments.tsx

```diff
@@ -57,6 +64,10 @@ export const Comments = ({ answerId, comments, statuses }: Props) => {
     open: false,
   });
 
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = React.useState(false);
+  const [commentToDelete, setCommentToDelete] =
+    React.useState<AnswerComments>();
```

Comment:
On peut simplifier ici en ne gardant que le state commentToDelete. Si il est undefined, c'est qu'il n'y a pas de commentaire à supprimer, sinon c'est que l'on a un commentaire à supprimer.

```suggestion
  const [commentToDelete, setCommentToDelete] =
    React.useState<AnswerComments | undefined>();
```